crates/red_knot_python_semantic/src/types.rs

```diff
@@ -175,6 +177,7 @@ impl<'db> Type<'db> {
                 // TODO raise error
                 Type::Unknown
             }
+            Type::BooleanLiteral(_) => Type::Unknown,
```

Comment:
Here we'll want to fall back to looking up attributes and methods on the `builtins.bool` class in our vendored typeshed stubs. But I think we can defer that for this PR.

Reply:
Yes, your understanding is correct there